core/src/style/pointStyle.h

```diff
@@ -50,6 +53,8 @@ class PointStyle : public Style {
 
     UniformLocation m_uTex{"u_tex"};
     UniformLocation m_uOrtho{"u_ortho"};
+
+    mutable std::unique_ptr<LabelMesh> m_mesh;
```

Comment:
I would prefer to find a suitable naming for StyledMesh and Tile::m_geometries instead that does not imply that styled data must be a mesh, or geometry. Text/SpriteLabels are containers for Text/SpriteLabel<s> 

TextStyleBucket (or -Data,-Container..) would be more appropriate naming when used for all StyledMesh implementations in my opinion.

For now, if you think that it really helps to have a hint that TextLabels implements (Styled)Mesh I would be ok to change it.